bootstrap.toml

```diff
@@ -0,0 +1,15 @@
+name = "Rust Bootstrap Team"
+description = "Meetings for the bootstrap team"
+
+[[events]]
+uid = "1705942319405"
+last_modified_on = "2024-01-22T18:00:00Z"
+title = "Rust Bootstrap Team Meeting"
+start = "2024-02-05T18:00:00Z"
```

Comment:
Which timezone does 18:00 refers to? 17:00 CET is equivalent to 16:00 UTC.

Reply:
Oops, my bad.
I did not check the output of Copilot, and it made sense in my head because timezones are confusing

Reply:
You can add the timezone field too.

Like here:
https://github.com/rust-lang/calendar/blob/14bb4ae00cc2cb9b152570bdb00b92ef3140c49b/infra.toml#L13

Reply:
Added new Europe/Paris TZ, and used it.